.github/workflows/web_app_deploy.yaml

```diff
@@ -22,6 +22,16 @@ on:
         type: boolean
         required: false
         default: true
+      health_check_path:
+        description: The health probe path exposed by the Web App.
+        type: string
+        default: ""
+        required: false
+      skip_swap_step:
```

Comment:
what about an opt-intinstead of opt-out?

Reply:
Fixed in [65c6818](https://github.com/pagopa/dx/pull/77/commits/65c6818bd914ae9bb60cc32ceada88b62996a63b)